locale/ja.po

```diff
@@ -24,10 +23,10 @@ msgid "increase-thumbnails-size"
 msgstr "ワークスペースのサムネイルを大きく表示"
 
 msgid "restore-thumbnails-background"
-msgstr "ワークスペースのサムネイルの壁紙を非表示"
+msgstr "ワークスペースのサムネイルに壁紙を表示"
 
 msgid "overview-firefox-pip"
-msgstr "アクティビティオーバービューにFirefoxのピクチャーインピクチャーを表示"
+msgstr "アクティビティオーバービューにFirefoxのピクチャーインピクチャーを非表示"
```

Comment:
This is the complete opposite of @axxapy 's description, which is `Display firefox's PIP window in the overview`.
This message is like `Hide Firefox's PIP window in the activity overview` if I translate it in English.

Reply:
@at-ishikawa Thank you very much

@oyajun By default, firefox's pip (picture in picture) window is not shown on the gnome overview. This setting enables mod that makes pip window visible in gnome's overview. Could you please review your translation?

Reply:
Sorry, I will fix it soon.
But even I turn this toggle switch on, firefox's pip window is not shown on the gnome overview.
Is this a bug?

Reply:
@oyajun 
It should work. I've just re-tested it on arch with latest gnome and firefox developer edition - works fine for me.
if it does not work for you, it would be great if you could debug it. 
File that is responsible for this functionality is located at the following location:
`~/.local/share/gnome-shell/extensions/gnome-ui-tune@itstime.tec/src/modFirefoxPipInOverview.js`